### DISCUSSION TITLE
Enter the key

### DISCUSSION THREAD

**Discussion by sanaousuke** [pjy612/SteamManifestCache#735] [Question / Answer]
ok because steamtools is down. i should use greenluma. how can i unlock the game using greenluma? because i'm using SAI it always says key is missing. how can i enter key decryption on SAI?

好的，因为 steamtools 已关闭。我应该使用greenluma。如何使用 greenluma 解锁游戏？因为我使用的是 SAI，它总是说密钥丢失。如何在 SAI 上输入密钥解密？

**Comment by pjy612** [comment]
手动把密钥写入到 config.vdf 之后 再使用 ，手动写入位置和格式 参考 wiki [DecryptionKey](https://github.com/pjy612/SteamManifestCache/wiki/DecryptionKey)
